CHANGELOG.md

```diff
@@ -0,0 +1,69 @@
+# Changelog
+
+### 0.9.1 (2015-07-20)
+
+* make default `controllerWhitelist` more specific
+
+---
+### 0.9.0 (2015-07-09)
+
+* add support for controller/action specific middleware
```

Comment:
It would be great to link to issues or commits here, to get more details about what sort of support has been added here.